docs/building-on-lisk/run-a-lisk-node.md

```diff
@@ -48,24 +48,22 @@ If you're looking to harden your app and avoid rate-limiting for your users, ple
 
 ## System requirements
 
-The following system requirements are recommended to run a Lisk L2 node.
+We recommend you the following hardware configuration to run a Lisk L2 node:
 
-### Memory
+- a modern multi-core CPU with good single-core performance
+- a minimum of 16 GB RAM (32 GB recommended)
+- a locally attached NVMe SSD drive
+- adequate storage capacity to accommodate both the snapshot restoration process (if restoring from snapshot) and chain data, ensuring a minimum of (2 * current_chain_size) + snapshot_size + 20%_buffer
+- if running with Docker, please install Docker Engine version [27.0.1](https://docs.docker.com/engine/release-notes/27.0/) or higher
 
-- Modern multi-core CPU with good single-core performance.
-- Machines with a minimum of 16 GB RAM (32 GB recommended).
-
-### Storage
-
-- Machines with a high performance SSD drive with at least 750GB (full node) or 4.5TB (archive node) free.
+*Note: If utilizing Amazon Elastic Block Store (EBS), ensure timing buffered disk reads are fast enough in order to avoid latency issues alongside the rate of new blocks added to Base during the initial synchronization process; io2 block express is recommended.*
```

Comment:
```suggestion
*Note: If utilizing Amazon Elastic Block Store (EBS), ensure timing buffered disk reads are fast enough in order to avoid latency issues alongside the rate of new blocks added to Base during the initial synchronization process; `io2 block express` is recommended.*
```